src/main/java/types/Host.java

```diff
@@ -250,6 +250,18 @@
      */
     Cpu cpu();
 
+    /**
+     * List of all host's CPUs with detailed information
+     * about the topology (socket, core) and with information
+     * about the current CPU pinning.
+     *
+     * @author Lucia Jelinkova <ljelinko@redhat.com>
```

Comment:
how about a sketch of the call and the return value as in the documentation of `katelloErrata`?